src/main.rs

```diff
@@ -41,6 +41,43 @@ fn convert_axis_angle_to_euler_xyz(rx: f64, ry: f64, rz: f64, angle: f64) -> (f6
 	return (euler_x, euler_y, euler_z);
 }
 
+/// Convert from Euler XYZ to AxisAngle.
+///
+/// All rotations are expected and returned as radians.
+///
+/// # Arguments
+///
+/// * `euler_rx` - The rotation around the X axis.
+/// * `euler_ry` - The rotation around the Y axis.
+/// * `euler_rz` - The rotation around the Z axis.
+///
+/// Known robots that use this format:
+/// - Staubli (NOTE: this example assumes radians values, but this robot uses degrees)
+fn convert_euler_xyz_to_axis_angle(euler_rx: f64, euler_ry: f64, euler_rz: f64) -> (f64, f64, f64, f64) {
+
+	let cx = cos(euler_rx/2.0);
```

Comment:
```suggestion
	let cx = cos(euler_rx / 2.0);
```

Let's stick to the normal formatting for operations.